lib/util/subprocess.py

```diff
@@ -3,21 +3,61 @@
 from lib import util
 
 
+class VerboseCalledProcessError(subprocess.CalledProcessError):
+    """
+    A CalledProcessError that includes captured stderr in its string representation.
+    """
+    def __str__(self):
+        base = super().__str__()
```

Comment:
```suggestion
        base = str(super())
```
?

Reply:
Alright, I forgot `super()` doesn't actually return the parent, but it instead creates a proxy object to access its attributes. Then your version is probably better.